src/test/kotlin/me/lensvol/blackconnect/BlackdClientTestCase.kt

```diff
@@ -85,4 +87,42 @@ class BlackdClientTestCase : BlackConnectTestCase() {
 
         Assert.assertEquals(blackdClient.blackdUrl, URL("http://localhost:45484"))
     }
+
+    private fun mockVersion(version: String) {
+        mockServer.apply {
```

Comment:
The issue is here: you are not calling `mockServer.reset()` before attaching a different response to the same request. When the test runs mock server will return an older response and that screws up your assertion.